File: app/src/main/java/com/odc/ticket/data/models/EvaluationModel.java
package com.odc.ticket.data.models;

import com.google.gson.annotations.SerializedName;

@SuppressWarnings({"unused","RedundantSuppression"})
public class EvaluationModel {

    @SerializedName("evaluationId")
    private long id;

    @SerializedName("transferId")
    private long transferId;

    @SerializedName("rating")
    private String rating;

    @SerializedName("note")
    private String note;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public long getTransferId() {
        return transferId;
    }

    public void setTransferId(long transferId) {
        this.transferId = transferId;
    }

    public String getRating() {
        return rating;
    }

    public void setRating(String rating) {
        this.rating = rating;
    }

    public String getNote() {
        return note;
    }

    public void setNote(String note) {
        this.note = note;
    }
}

File: app/src/main/java/com/odc/ticket/data/models/room/EvaluationModel.kt
package com.odc.ticket.data.models.room

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.SerializedName

@Entity
data class EvaluationModel(
    @PrimaryKey(autoGenerate = true) val evaluationId: Int,
    @ColumnInfo(name = "transferId") val transferId: Int,
    @ColumnInfo(name = "currencyName") val currencyName: String,
    @ColumnInfo(name = "rating") val rating: String,
    @ColumnInfo(name = "note") val note: String,
)